examples/pytorch/torchscript/IrisClassification/iris_classification.py

```diff
@@ -0,0 +1,104 @@
+# pylint: disable=W0221
+# pylint: disable=W0613
+# pylint: disable=W0223
```

Comment:
why ignore these pyling rules?

Reply:
`W0223: Method '_forward_unimplemented' is abstract in class 'Module' but is not overridden (abstract-method)` - Eventhough the forward method is implemented in IrisClassifier class, pylint throws the error. Hence suppressed W0223. 

i guess W0221 and W0613 are not need for this example. Mostly on pytorch lightning examples, batch_idx would be added as part of training step and since its not used the pylint error is thrown. We cannot remove the batch_idx argument as it will result in non matching abstract method signature. 

But for this example W0221 and W0613  can be removed i believe, i will add a new commit for this change.